src/cloud-cli/userdb.ts

```diff
@@ -168,17 +169,14 @@ export function migrate(): number {
     }
   }
 
-  let dbType = configFile.database.user_dbclient;
-  if (dbType === undefined) {
-    dbType = "knex";
-  }
-
-  const migratecommands = configFile.database.migrate;
+  const dbType = configFile.database.user_dbclient || UserDatabaseName.KNEX;
+  const migrationScript = `node_modules/.bin/${dbType}`
+  const migrationCommands = configFile.database.migrate;
 
   try {
-    migratecommands?.forEach((cmd) => {
-      const command = "npx " + dbType + " " + cmd;
-      logger.info("Executing " + command);
+    migrationCommands?.forEach((cmd) => {
+      const command = `node ${migrationScript} ${cmd}`
+      logger.info(`Executing migration command: ${command}`);
```

Comment:
@manojdbos @kraftp something to document is that we expect the user database ORM to be the same than the one used for migrations. I trust this is the common path but users could want a separate behavior. So we should document it.